client/app/telecom/telephony/line/phone/order/translations/Messages_fr_FR.xml

```diff
@@ -25,7 +25,7 @@
    <translation id="telephony_line_phone_order_process_exchange2" qtlid="459040">Vous payez la caution du nouveau matériel lors de la commande. Celle de l'ancien matériel vous est restituée sous 10 jours après réception de l'équipement par nos services.</translation>
    <translation id="telephony_line_phone_order_return_phone" qtlid="459053">Renvoyer mon équipement actuel ({{ phone }})</translation>
    <translation id="telephony_line_phone_order_return_address_title" qtlid="459066">Adresse de retour :</translation>
-   <translation id="telephony_line_phone_order_return_address_content" qtlid="459079">OVH SAS<br />Service Après Vente<br />13 rue Kellermann<br />59100 Roubaix</translation>
+   <translation id="telephony_line_phone_order_return_address_content">OVH SAS<br />Service Après Vente<br />155 avenue Georges Hannart<br />59170 Croix</translation>
```

Comment:
Could the `<br/>` be removed and the translation split into chunks ?

Reply:
Please check if this is okay ? If `<br>` has to be avoided altogether, I can add a class and `<p>` tag